extensions/open-finder-selection-with/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "open-finder-selection-with",
+  "title": "Open Finder Selection With",
+  "description": "Filter an application to open selected Finder items.",
+  "icon": "Finder.icns",
```

Comment:
Please, use `.png` here as it described in the [Guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)